modules/http2/src/main/java/org/glassfish/grizzly/http2/Http2ServerFilter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2015, 2019 Oracle and/or its affiliates and others
```

Comment:
Shouldn't this keep the "All rights reserved." part, but just moves it to next line?

Reply:
I was under the impression that the `All rights reserved.` section wasn't required as the EPL implies it? Would you be able to show me an example of the format you'd like?

Reply:
Maybe this one?

https://github.com/eclipse-ee4j/websocket-api/blob/master/api/server/src/main/java/javax/websocket/server/HandshakeRequest.java#L2